python/cog/server/runner.py

```diff
@@ -316,7 +316,16 @@ def _predict(
 
     for k, v in input_dict.items():
         if isinstance(v, types.URLPath):
-            input_dict[k] = v.convert()
+            try:
+                input_dict[k] = v.convert()
+            except Exception as e:
```

Comment:
I think we probably want a more tightly scoped exception here, to match what we do with webhooks and in director.

```suggestion
            except requests.exceptions.RequestException as e:
```

Reply:
I just tried a couple of different things to try to break it, and Requests seems to do a good job of re-wrapping exceptions—so doesn't appear to be a problem, at least in some light testing. I still suspect we might hit exceptions not wrapped here, but it might be the better trade-off for now